src/pip/_vendor/urllib3/packages/ssl_match_hostname/_implementation.py

```diff
@@ -11,7 +11,7 @@
 # python-3.5) otherwise only do DNS matching.  This allows
```

Comment:
This comment needs updating?

Reply:
This comment is a part of the vendored urllib3, not pip. Is it customary to make such a change?

https://github.com/urllib3/urllib3/blob/fd0c475cc2c51aedb6c89d7b9be58d850966ee6a/src/urllib3/packages/ssl_match_hostname/_implementation.py#L9-L16

FWIW, I did open a PR upstream to help change that there: https://github.com/urllib3/urllib3/pull/2122

Reply:
Oooo. I should read more carefully! :)